docs/guides/cli/basics.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 id: cli-basics
 title: Basic Functionality
+sidebar_position: 5
```

Comment:
Changing file content is better to review than changing file names IMO. I can also revert this, but we did not use the number convention elsewhere so this is also more consistent.

Reply:
Changing filenames broke two relative links which causes the website build to fail. Logs: 

```
Error: Docs markdown link couldn't be resolved: (../guides/cli/10_config-with-cli.mdx) in /docs/docs/concepts/personal-access-token.mdx for version current
    at Array.forEach (<anonymous>)
Error: Docs markdown link couldn't be resolved: (./cli/20_proxy-and-tunnel.mdx) in /docs/docs/guides/custom-domains.mdx for version current
    at Array.forEach (<anonymous>)
```

Reply:
The system where the document order is determined by filenames is something I'm trying out and I really like it so far. The benefits are: 
- the `sidebars` file is leaner
- you can see the correlation between the left navigation and the files in the directory at the first glance
- when you start messing with the file order you don't have to open any other files to know what to do - everything is visible at a glance without having to inspect the metadata section of the docs

I'm slowly implementing this system as I update/refactor/create new documents. For know, I think that this is the way to go. 

Please revert filenames to their initial state. This will fix your failing build.